src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const clickPromis = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
+
+const div = document.createElement('div');
+
+document.body.append(div);
+
+clickPromis
```

Comment:
The variable name 'clickPromis' is a bit misleading. It's not clear what it's for, and it's not correctly spelled. Consider renaming it to something more descriptive and correct, like 'clickPromise'.